cmd/gencerts_test.go

```diff
@@ -0,0 +1,89 @@
+package cmd_test
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/OpenBazaar/openbazaar-go/cmd"
+)
+
+func TestGenCertsGenericDefaults(t *testing.T) {
+
+	if dataDir, err := ioutil.TempDir("", "gencerts_test"); err != nil {
```

Comment:
We have a schema manager which helps with this stuff already. Take a look at https://github.com/OpenBazaar/openbazaar-go/blob/master/repo/db/chat_test.go#L13-L20 and replicate the pattern there.